app/bundles/EmailBundle/Translations/en_US/messages.ini

```diff
@@ -470,3 +409,8 @@ mautic.email.stat.tooltip="Details may not match summary numbers if the contact
 mautic.email.stat.simple.tooltip="Details may not match summary numbers if the contact no longer exists in your Mautic Account"
 mautic.email.associated.contacts="Contacts"
 mautic.email.segment.choice.generated_email_domain="Email Domain"
+mautic.email.config.mailer_encryption.none="None"
+mautic.email.config.mailer_encryption.ssl="SSL"
+mautic.email.config.mailer_encryption.tls="TLS"
+mautic.email.config.dsn_error="You have an invalid DSN set in your configuration. Please update it to a valid DSN."
```

Comment:
Will people know what a DSN is? I think it would be helpful to point them to a KB article here which explains what it means and how to fix it. Can we do that?

Reply:
This will change today. See https://github.com/mautic/mautic/pull/11613#issuecomment-1591120034

Reply:
Okie doke - I think we do need to assume that folks won't know all the complicated terminology associated with email transports and whatnot so the more we can help them the better :)